frontend/store/index.js

```diff
@@ -19,6 +19,12 @@ export const actions = {
     if (resp) {
       commit("ratings/SETNBHITS", resp.data.nbHits);
       commit("ratings/SETCOMPANIES", resp.data.hits);
+      commit("ratings/SETAVERAGE", resp.data)
+      let average = 0;
+      for (let item = 0; item < resp.data.hits.length; item++) {
+        average += resp.data.hits[item].salary
+        commit("ratings/SETAVERAGE", average / resp.data.hits.length);
+      }
```

Comment:
There is no need to do this. The average is already calculated for you in the backend you need to give it the appropriate filter and it will work. Like it is done on this PR https://github.com/osscameroon/camerdevs/pull/80#issue-1133875128